api/src/main/java/org/pmiops/workbench/api/ClusterController.java

```diff
@@ -199,10 +199,7 @@
         .forEach(
             notebook ->
                 userRecentResourceService.updateNotebookEntry(
-                    workspaceId,
-                    userProvider.get().getUserId(),
-                    gcsNotebooksDir + "/" + notebook,
-                    now));
+                    workspaceId, userProvider.get().getUserId(), gcsNotebooksDir + "/" + notebook));
```

Comment:
nits: did spotless do this? FWIW if it's more than two nontrivial arguments I prefer them one to a line.

Also seems like we should have a full notebook path function somewhere handy so this function doesn't need to know what separator to use or do any string manipulation in-place.

Reply:
yeah. it's enforcing this style so I think I'm going to stick with it.